### PR TITLE
Update scss-lint to 0.42.2

### PR DESCRIPTION
**Pull request by cole-sanderson**
Status: **Ready for Review**
Owner: @cole-sanderson
Reviewers: @jeffkamo @ry5n 

## Changes
- Update scss-lint to 0.42.2
- Update some comments 

## Todos:
- [x] Developer (@jeffkamo)
- [x] Developer (@yourpalsonja)

### Feedback:
_none so far_

### CODE REVIEW

File: css/.scss-lint.yml [hunk 1]
```diff
@@ -219,13 +220,19 @@ linters:
     SpaceAfterVariableName:
         enabled: true
 
+    # Checks for space around operators on values.
+    # Configure: `one_space` or `no_space`
+    SpaceAroundOperator:
+        enabled: true
+        style: one_space
```
Comment (jeffkamo) [inline css/.scss-lint.yml]: This is for things like `$unit*2` vs `$unit * 2`?

Do we have a preference for one over the other?

Comment (cole-sanderson) [inline css/.scss-lint.yml]: Yes that's correct. 

Hmm... Good question, I usually don't add space. 

Maybe we should change to `no_space` ?

**Comment by jeffkamo** [comment]
+1

**Comment by yourpalsonja** [comment]
:+1: LGTM!

**Pull request merged**